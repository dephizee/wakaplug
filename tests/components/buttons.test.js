import React from 'react';
import renderer from 'react-test-renderer';
import { ButtonStyled} from "./../../components/buttons.style";


test('ButtonStyled working', () => {
  const component = renderer.create(
    <ButtonStyled>Hello Guys</ButtonStyled>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});