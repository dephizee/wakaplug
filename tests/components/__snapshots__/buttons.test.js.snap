// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonStyled working 1`] = `
<button
  className="sc-bdvvtL jdwahO"
>
  Hello Guys
</button>
`;
